lib/si-runtime-rs/src/lib.rs

```diff
@@ -18,13 +21,16 @@ const COMPUTE_EXECUTOR_THREAD_PRIORITY: u8 = 25;
 // Tokio runtime shutdown timeout for compute executors
 const COMPUTE_EXECUTOR_TOKIO_RT_SHUTDOWN_TIMEOUT: Duration = Duration::from_secs(60 * 10);
 
+const MAX_WORKER_THREADS: usize = 32;
```

Comment:
Why 32?

Reply:
Oh, this came from the blog post I was using and I think it's kind of on the arbitrary side.